fastlane/lib/fastlane/actions/crashlytics.rb

```diff
@@ -41,12 +41,16 @@ def self.description
       end
 
       def self.available_options
+        platform = Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
+        ipa_path_default = (!platform || platform == :ios) && Dir["*.ipa"].last
```

Comment:
Can we move this to multiple lines? For non-Ruby developers the `&&` operator may not be 100% clear